src/test/java/core/basesyntax/service/FruitTransactionTest.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.FruitTransaction;
+import org.junit.jupiter.api.Test;
+
+class FruitTransactionTest {
+    @Test
+    void setOperation_setsOperationCorrectly() {
```

Comment:
lets apply method naming convention
`<method under test>_<state>_<expected result>`
```suggestion
    void setOperation_supply_ok() {
```